modules/core/src/test/java/org/apache/ignite/platform/PlatformNullablePrimitivesTask.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
```

Comment:
I believe, we should put actual date in new files?
```suggestion
 * Copyright 2023 GridGain Systems, Inc. and Contributors.
```